src/main/java/Duke/Task/Task.java

```diff
@@ -0,0 +1,29 @@
+package Duke.Task;
+
+
+public abstract class Task {
+    protected String description;
+    protected boolean done;
+
+    public Task(String description){
+        this.description=description;
+        this.done=false;
```

Comment:
Try to leave some spaces, in general, like so to improve readability:
`this.description = description;`